tests/fsharp/tests.fs

```diff
@@ -864,6 +853,28 @@ module CoreTests =
             testOkFile.CheckExists()
         end
 
+    // This test stays in FsharpSuite for a later migration phases, it uses hardcoded #r to a C# compiled cslib.dll inside
```

Comment:
Does this still apply then?

Reply:
I have no idea, the comment was there before :)

Reply:
No I know, but from what I understand - you are dehardcoding this cslib.dll thing hence the comment might not apply anymore :)

Reply:
Yes and no, there in fact is a hardcoded #r inside.

Reply:
I see. Well I guess this can go anyway, but can probably stay either, doesn't matter.